.buildkite/pipeline.yml

```diff
@@ -4,8 +4,8 @@ agents:
   queue: agent-runners-linux-amd64
 
 steps:
-  - name: ":go: go fmt"
-    key: test-go-fmt
+  - name: ":go: Check go-fmt and go-mod-tidy"
```

Comment:
What do you think about combining this check with the generated files check?

Reply:
It's a bit risky b/c the check for `gofmt` does not use git to check if the working tree is clean, but the check for `go generate` does. So the earlier test could pass but leave the working tree dirty, which will be a false positive for the latter test. I don't think that condition is activated, but I think it's safer to make all 3 checks assert that they are no-ops on the working directory (for committed files).